webpush/encrypt.go

```diff
@@ -17,14 +17,13 @@
 //
 // Sending a message:
 //   import (
-//     "strings"
 //     "github.com/googlechrome/push-encryption-go/webpush"
 //   )
 //
 //   func main() {
 //     // The values that make up the Subscription struct come from the browser
 //     sub := &webpush.Subscription{endpoint, key, auth}
-//     webpush.Send(nil, sub, "Yay! Web Push!", nil)
+//     webpush.Send(nil, sub, "Yay! Web Push!", "")
```

Comment:
It's best to use ExampleXxx tests exactly for this purpose: make sure examples compile.
https://golang.org/pkg/testing/#hdr-Examples